grab_disks

```diff
@@ -71,8 +71,35 @@ fi
 # to make sure the disk exist.
 #
 if [ $numb_disks -eq 0 ]; then
-	echo  "Need to have disks to do fio"
+	echo  "Need to have disks to perform the test"
 	exit 1
 fi
+if [ -f "/tmp/meta_data.yml" ]; then
+	grep -q Azure /tmp/meta_data.yml
+	#
+	# Only for Azure.
```

Comment:
Instead of just this, could you expand a little so someone coming along later has an idea of why this is being done?  You've explained the issue to me offline but someone else later won't have that context and this will seem kinda arbitrary.